Vizzuality/half-earth-v3
# Feature/challenges sidebar

## Challenges and Ranking sidebars
### Description
On this PR new sidebar content is created for Challenges and Ranking view inside National Report Cards.
The header for all the sidebars has been updated to match the designs (integrating the tabs component).
The information cards display data coming from the metadata service.
There are a couple of things missing that will be adressed in a separate PR:
- Close circular button.
- Share and download buttons.
### Designs
[Challenges sidebar](https://invis.io/FH108BHHVCQ4#/445081449_sidebar_Challenges)
[Ranking sidebar](https://invis.io/FH108BHHVCQ4#/445090343_sidebar_Ranking)
### Testing instructions
Go to [a National Report Card](http://localhost:3000/dataGlobe?globe=eyJjZW50ZXIiOls4LjA4MTY2MzI1ODUwOTYzOCwxNy43OTQ0NDEzMDY5MjM1MDZdLCJ6b29tIjo2LjUzMTkzNzE4OTQ1NzczOCwiY291bnRyeUlTTyI6Ik5FUiIsImNvdW50cnlOYW1lIjoiTmlnZXIiLCJsYW5kc2NhcGVWaWV3IjpmYWxzZSwiYWN0aXZlTGF5ZXJzIjpbeyJ0aXRsZSI6ImdyYXBoaWNfbGF5ZXIifSx7InRpdGxlIjoibWFzay1sYXllciJ9LHsidGl0bGUiOiJjaXRpZXNfbGFiZWxzX2xheWVyIn0seyJ0aXRsZSI6ImxhbmRzY2FwZV9mZWF0dXJlc19sYWJlbHNfbGF5ZXIifSx7InRpdGxlIjoiY291bnRyeV9wcmlvcml0eV9sYXllciIsIm9wYWNpdHkiOjAuNH0seyJ0aXRsZSI6Im1lcmdlZC1wcm90ZWN0ZWQiLCJvcGFjaXR5IjowLjR9XX0=&ui=eyJzY2VuZU1vZGUiOiJsb2NhbCJ9) and check the sidebar :)
### Feature relevant tickets
Challenges: https://half-earth-map.atlassian.net/browse/HE-270
Ranking: https://half-earth-map.atlassian.net/browse/HE-271
